Content.Server/SurveillanceCamera/Systems/SurveillanceCameraMapSystem.cs

```diff
@@ -0,0 +1,88 @@
+using System.Numerics;
+using Content.Server.Power.Components;
+using Content.Shared.DeviceNetwork.Components;
+using Content.Shared.SurveillanceCamera.Components;
+
+namespace Content.Server.SurveillanceCamera;
```

Comment:
Should probably be `Content.Server.SurveillanceCamera.Systems`

Reply:
Ah, silly. Suppose it's fine then; one of those things that can be fixed in a whole-system cleanup/refactor/whathaveyou.